### ISSUE TITLE
Why is my accuracy so low?

### ISSUE THREAD

**Issue by sugy0114** [Mabel0403/CAMP#3]
Why is my accuracy so low?Looking forward to your reply
------------------------------[Evaluate]------------------------------
Extract Features:
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 296/296 [04:05<00:00,  1.21it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 8/8 [00:07<00:00,  1.01it/s]
Compute Scores:
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 37855/37855 [00:12<00:00, 3007.35it/s]
Recall@1: 5.8988 - Recall@5: 16.8828 - Recall@10: 25.2014 - Recall@top1: 26.5223 - AP: 9.1835

**Comment by Mabel0403** [comment]
Hi sugy0114, could you please specify the dataset used for evaluation and the trained model applied?

**Comment by sugy0114** [comment]
> Hi sugy0114, could you please specify the dataset used for evaluation and the trained model applied?

Hello, my training model is set up like this
@register_model
def convnext_base(pretrained=True, in_22k=True, **kwargs):
    model = ConvNeXt(depths=[3, 3, 27, 3], dims=[128, 256, 512, 1024])
    if pretrained:
        url = model_urls['convnext_base_22k'] if in_22k else model_urls['convnext_base_1k']
        state_dict = torch.load("/slurm-files/sgy/code/Sample4Geo/pretrained/university/convnext_base_22k_1k_224.pth")
        print("111111111111")
        model.load_state_dict(state_dict,strict=False)
        print("---------base----------")
        '''
        checkpoint = torch.hub.load_state_dict_from_url(url=url, map_location="cpu")
        print(url)
        model.load_state_dict(checkpoint["model"], strict=False)
        '''
    return model

**Comment by Mabel0403** [comment]
Hi sugy, 

I recommend loading the model as mentioned in #4 . If your accuracy remains low, please go through the following checklist:

1. Use my pretrained model for U-1652: [https://drive.google.com/file/d/1qHjXr3VVQuJZ5kE5u7YrUB8id90Nv2GJ/view?usp=sharing] **directly for evaluation**. This will help verify if there are issues with your data or evaluation code.
2. Double-check your training configuration settings.

If the problem persists, please provide more details about your training process, and I’ll try to help further.
